api/src/pcapi/core/users/factories.py

```diff
@@ -171,8 +173,19 @@ def beneficiary_fraud_checks(
     ) -> list[fraud_models.BeneficiaryFraudCheck]:
         import pcapi.core.fraud.factories as fraud_factories
 
+        obj.email = f"{obj.firstName}.{obj.lastName}.{obj.id}@example.org".lower()
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** ‏C'est l'occaz de différencier les users générés
genre `@passculture.gen` ou que sais-je

low logaf :smile:

Reply:
🤓
J'y avais pas pensé. J'aime bien l'idée ! low logaf certes, mais si rapide. Vas pour @passculture.gen